DuckDuckGo/SiteRatingView.swift

```diff
@@ -51,7 +51,6 @@ public class SiteRatingView: UIView {
 
     public override func layoutSubviews() {
```

Comment:
This method is no longer needed, can we delete it?

Reply:
Thanks for pointing that out, done 👍